spec/takeaway_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'takeaway'
+describe Takeaway do
+  subject { Takeaway.new }
+  let(:dish) { double("Chicken") }
```

Comment:
So it's not necessary to use doubles for simple strings (like `"Chicken"`), because we use doubles for objects on which we want to "mock" or "fake" method calls — in this file, you could simply replace the double by a plain string value:
```ruby
let(:dish) { "Chicken" }
```

For example, let's say you've extracted the `Menu` class (as explained in my other comment above) — you would need a new tests file for this new class (likely named `menu_spec.rb`), and then in the `Takeaway` tests, you would need to use a double to remove the direct dependency that `Takeaway` has on `Menu`. This would be a step further from extracting the `Menu` class, so work on this only once the Menu class has been extracted correctly :)